Assets/HoloToolkit-Examples/Input/Scripts/TestButton.cs

```diff
@@ -44,12 +46,15 @@ public bool Focused
                 if (focused != value)
                 {
                     focused = value;
+                    //Calling to UpdateButtonAnimation() method 
                     UpdateButtonAnimation();
                 }
             }
         }
 
         private bool stayFocused;
+
+        //Property of stayFocused
```

Comment:
Usually we use xml summary tags for documentation. Here's a good example:

```
/// <summary>
/// My Documentation will go here.
/// </summary>
```
This allows others to read the description of your property in the tooltip

Reply:
What I did was what we usually did in our school project. Thanks for letting me know to be more efficient!!